### ISSUE TITLE
[Regression] Gamepad missing entry in Wine Control Panel

### ISSUE THREAD

**Issue by Snaggly** [lutris/wine#79]
Hello I noticed my XInput and DInput controllers are can't be detected within games that don't use DXVK, (eg. OpenGL or use native D3DX) on Lutris-Wine 6.0 and higher. The controllers were perfectly usable up to 5.7-11. 

Tested games: Hyperdimension Neptunia Re;Birth 1-3, Rayman Origins, Rayman 2 and 3, Jazz Jackrabbit 2.

The controllers apparently only work with games that run on DirectX 11 and 12 with VKD3D (D3D10 untested)

The vanilla Wine build (6.7) was working fine, Wine-TKG 6.1 from the Frogging-Family is working fine too and the latest Proton 6.3 and Proton-GE 6.8 can find my controller with those few games too. So I'm suspecting something along this build must have gone wrong.

**Comment by dpanter** [comment]
Nuclear Throne, Dinput Logitech F710, Debian sid, Lutris 6.4 works. I don't have those other games tested but this is a DX9 game.
[System info + Lutris log](https://pastebin.com/LtJc1qdt)

**Comment by Snaggly** [comment]
Update:
On other Wine/Proton versions I found on the Wine Control Panel->Gamepad Settings these two entries.
![Screenshot_20210607_185039](https://user-images.githubusercontent.com/1353354/121060155-8ee6b800-c7c2-11eb-8b24-a192773d54f2.png)

On Lutris-Wine 6.0 and higher the first active entry (event) one is just gone. However when activating the js one below I was able to finally use my gamepad again. This setting is not persistent tho, and I did not test a lot with that as the game just lagged a lot and crashed right after 😆.
![Screenshot_20210607_185443](https://user-images.githubusercontent.com/1353354/121061493-30bad480-c7c4-11eb-90cc-b838d34054ca.png)
![Screenshot_20210607_184451](https://user-images.githubusercontent.com/1353354/121061535-403a1d80-c7c4-11eb-9a33-8db8ecc1f9f9.png)
